molo/yourwords/views.py

```diff
@@ -0,0 +1,7 @@
+from django.views.generic.edit import CreateView
+from molo.yourwords.forms import CompetitionEntryForm
+
+
+class CompetitionEntry(CreateView):
```

Comment:
Also can you call this `CompetitionEntryView`, just to make sure it's clear from the name